src/styles/input.scss

```diff
@@ -8,6 +8,8 @@ $block: #{$fd-namespace}-input;
 
 .#{$block} {
   @include fd-form-text();
+  @include fd-form-states();
```

Comment:
mixin `fd-form-text` in doing the same thing, so you can remove it

Reply:
the fd-form-states gives the fiori semantic inputs border color.
it also adds the box shadow around the input for horizon styles when you hover over it. 

These 2 were separated because the standonline tokenizer does not seem do have any interactions involving hover/focus as per the spec. (It uses the parent component's interactions when used within input-group)